Proiect-JS-main/src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Calculator corporal</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <h2>
+        <span>Calculator corporal Kg -> Lbs, Cm -> In</span>
+      </h2>
+      <div class="wrapper-1">
+        <label for="weight">Greutate in Kilograme:</label>
+        <input type="number" id="ratio-weight" value="80" />
```

Comment:
Valoarea din `for` trebuie să se potrivească id-ului elementului pentru care pui label.

```suggestion
        <label for="ratio-weight">Greutate in Kilograme:</label>
        <input type="number" id="ratio-weight" value="80" />
```